bokehjs/src/coffee/api/typings/models/scales.d.ts

```diff
@@ -0,0 +1,3 @@
+declare namespace Bokeh {
+
+}
```

Comment:
There are no typings for transforms yet, so this has to be added first. Otherwise changes should be straightforward (move `mappers.d.ts -> scales.d.ts`, change the base and do renaming).